wear/src/screenshotTest/java/android/os/Build.java

```diff
@@ -0,0 +1,10 @@
+package android.os;
+
+public class Build {
+    // On screenshot-0.0.1-alpha11 FINGERPRINT is not set because of that we need to set everything
+    // For more details: https://issuetracker.google.com/u/2/issues/422488144#comment17
```

Comment:
```suggestion
    // For more details: https://issuetracker.google.com/issues/422488144#comment17
```

Clean link, `u/2/` is personal and unstable